index.js

```diff
@@ -667,43 +673,43 @@ class Plugin extends MiniAccountsPlugin {
         Buffer.from(account.getPaychan().publicKey.substring(2), 'hex')
       )
     } catch (err) {
-      debug('verifying signature failed:', err.message)
+      this._log.debug('verifying signature failed:', err.message)
     }
 
     // TODO: better reconciliation if claims are invalid
     if (!valid) {
-      debug(`got invalid claim signature ${signature} for amount ${dropAmount} drops`)
+      this._log.debug(`got invalid claim signature ${signature} for amount ${dropAmount} drops`)
```

Comment:
this will really just happen if someone modified their asym-client code or if there's a bug in the code. It would be good to see this on the `error` level